src/Migrations/M022UpdatePostRevisions.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+
+/**
+ * Update the limit of post revisions.
+ */
+class M022UpdatePostRevisions extends MigrationScript
+{
+    /**
+     * Perform the actual migration.
+     *
+     * @param MigrationRecord $record Information on the execution, can be used to add logs.
+     * phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter -- interface implementation
+     */
+    protected static function execute(MigrationRecord $record): void
+    {
+        global $wpdb;
+
+        if (!get_option('revisions_to_keep')) {
+            // If not, set 20 revisions by default
+            $MAX_REVISIONS = 20;
+            update_option('revisions_to_keep', $MAX_REVISIONS);
+        } else {
+            $MAX_REVISIONS = get_option('revisions_to_keep');
+        }
+
+        $wpdb->query('SET @rownumber:=0');
+        $wpdb->query('SET @postparent:=NULL');
+        $sql = '
```

Comment:
In case we want upgrade to `MySQL 8.0`, it's better if we implement the `ROW_NUMBER` function.
```sql
DELETE FROM wp_posts
            WHERE ID IN (
                SELECT ID
                FROM (
                    SELECT
                        post_parent,
                        ID,
                        post_date,
                        row_number() OVER (PARTITION BY post_parent ORDER BY post_date desc) AS rn
                    FROM (
                        SELECT
                            ID,
                            post_parent,
                            post_date
                        FROM wp_posts
                        WHERE post_parent IN (
                            SELECT post_parent
                            FROM wp_posts
                            WHERE post_type = "%2$s"
                            GROUP BY post_parent
                            HAVING count(ID) > %1$s
                        ) AND post_type = "%2$s"
                        ORDER BY post_date ASC
                    ) AS r
                ) AS d WHERE d.rn > %1$s
```